include/zephyr/net/socket_ncs.h

```diff
@@ -34,11 +34,39 @@ extern "C" {
  */
 #define TLS_DTLS_HANDSHAKE_TIMEO 14
 
+/** Socket option to set and get DTLS CID setting, specifically for nRF sockets.
+ *  The option accepts an integer, indicating the setting.
+ *  Accepted vaules for the option are: 0, 1 and 2.
+ */
+#define TLS_DTLS_CID 17
+
+/** Socket option to get DTLS CID status, specifically for nRF sockets.
+ *  The option accepts an integer, indicating the setting.
+ *  Accepted vaules for the option are: 0, 1 and 2.
```

Comment:
```suggestion
 *  Accepted vaules for the option are: 0, 1, 2 and 3.
```

Reply:
We might also just remove this sentence, as the options are given below. Same with the CID and handshake status.